cumulus_library/cli.py

```diff
@@ -90,21 +108,23 @@ def clean_study(
 
         for target in targets:
             if prefix:
-                manifest_parser = study_parser.StudyManifestParser()
+                manifest = study_parser.StudyManifestParser()
+                schema = self.get_schema(manifest)
                 cleaner.clean_study(
-                    manifest_parser=manifest_parser,
+                    manifest_parser=manifest,
                     cursor=self.cursor,
-                    schema_name=self.schema_name,
+                    schema_name=schema,
                     verbose=self.verbose,
                     stats_clean=stats_clean,
                     prefix=target,
```

Comment:
This happened a bunch in here - I'm not moving things around because i'm trying to not make a breaking change, but this was a headache and i think this might be the last major feature before i make breaking changes.